models/issue_watch.go

```diff
@@ -97,14 +100,11 @@ func getIssueWatchers(e Engine, issueID int64, listOptions ListOptions) (watches
 }
 
 func removeIssueWatchersByRepoID(e Engine, userID int64, repoID int64) error {
-	iw := &IssueWatch{
-		IsWatching: false,
-	}
 	_, err := e.
 		Join("INNER", "issue", "`issue`.id = `issue_watch`.issue_id AND `issue`.repo_id = ?", repoID).
 		Cols("is_watching", "updated_unix").
```

Comment:
If we're not updating, what do we need the `Cols()` list for?

Reply:
good point!